CHANGELOG.md

```diff
@@ -9,6 +9,63 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
+- The `WordPress.Arrays.ArrayKeySpacingRestrictions` sniff has two new errorcodes: `TooMuchSpaceBeforeKey` and `TooMuchSpaceAfterKey`. Both auto-fixable.
+    The sniff will now check that there is _exactly_ one space on the inside of the square brackets around the array key for non-string, non-numeric array keys. Previously, it only checked that there was whitespace, not how much whitespace.
+- `WordPress.Arrays.ArrayKeySpacingRestrictions`: the fixers have been made more efficient and less fixer-conflict prone.
+- `WordPress.NamingConventions.PrefixAllGlobals`: plugin/theme prefixes should be at least three characters long. A new `ShortPrefixPassed` error has been added for when the prefix passed does not comply with this rule.
+- `WordPress.WhiteSpace.CastStructureSpacing` now allows for no whitespace before a cast when the cast is preceded by the spread `...` operator. This pre-empts a fixer conflict for when the spacing around the spread operator will start to get checked.
+- The `WordPress.WP.DeprecatedClasses` sniff will now detect classes deprecated in WP 4.9 and WP 5.3.
+- The `WordPress.WP.DeprecatedFunctions` sniff will now detect functions deprecated in WP 5.3.
+- `WordPress.NamingConventions.ValidHookName` now has "cleaner" error messages and higher precision for the line on which an error is thrown.
+- `WordPress.Security.EscapeOutput`: if an error refers to array access via a variable, the array index key will now be included in the error message.
+- The processing of the `WordPress` ruleset by `PHP_CodeSniffer` will now be faster.
+- Various minor code tweaks and clean up.
+- Various minor documentation fixes.
+- Documentation: updated the repo URL in all relevant places.
+
+### Deprecated
+- The `WordPress.WP.TimezoneChange` sniff. Use the `WordPress.DateTime.RestrictedFunctions` instead.
+    The deprecated sniff will be removed in WPCS 3.0.0.
+
+### Fixed
+- All sniffs in the `WordPress.Arrays` category will no longer treat _short lists_ as if they were a short array.
+- The `WordPress.NamingConventions.ValidFunctionName` and the `WordPress.NamingConventions.PrefixAllGlobals` sniff will now ignore functions marked as `@deprecated`.
+- Both the `WordPress.NamingConventions.PrefixAllGlobals` sniff as well as the `WordPress.WP.GlobalVariablesOverride` sniff have been updated to recognize variables being declared via (long/short) `list()` constructs and handle them correctly.
+- Both the `WordPress.NamingConventions.PrefixAllGlobals` sniff as well as the `WordPress.WP.GlobalVariablesOverride` sniff will now take a limited list of WP global variables _which are intended to be overwritten by plugins/themes_ into account.
+    Initially this list contains the `$content_width` and the `$wp_cockneyreplace` variables.
```

Comment:
```suggestion
    Initially this list contained the `$content_width` and the `$wp_cockneyreplace` variables.
```

Reply:
@GaryJones Second opinion please.

IMO the present tense is correct as the list has not changed since the fix this changelog entry refers to, was pulled and merged and this is the first mention of this list in the changelog.

Reply:
Agreed - `contains`, as it still talking in the present - "...sniff will NOW take a limited list...[and that list] contains...".